manager/manager.go

```diff
@@ -82,6 +82,9 @@ type Options struct {
 	SignRequest        SignRequestFunc
 	WriteKeypair       WriteKeypairFunc
 	ReadyToRequest     ReadyToRequestFunc
+
+	// BackoffConfig configures the exponential backoff applied to certificate renewal failures.
+	BackoffConfig *wait.Backoff
```

Comment:
This is more explicit. I would always have to refer the field comment without `Renewal` prefix. 

```suggestion
	// RenewalBackoffConfig configures the exponential backoff applied to certificate renewal failures.
	RenewalBackoffConfig *wait.Backoff
```